package.json

```diff
@@ -140,5 +141,15 @@
       "cp -af /usr/lib/PSAMMEAD-COMPONENT-LIBRARY/bake-scripts/. /etc/bake-scripts/PSAMMEAD-COMPONENT-LIBRARY",
       "chmod +x /etc/bake-scripts/PSAMMEAD-COMPONENT-LIBRARY/*"
     ]
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.css": [
```

Comment:
Simorgh uses css-in-js so we have no `.css` file to lint. Have a look at the `test:lint:css` script we already have and make sure stylelint is run against those same files.

Reply:
Done